src/styles/variables.scss

```diff
@@ -0,0 +1,12 @@
+$clock-width: 500px;
+$clock-height: 500px;
+$main-height: 100%;
```

Comment:
Seems that `$main-height` is a useless variable

Reply:
why useless ? I used it for positioning in the center in "calc"